From af5e83142f64b0f8ba0df4203d3da8175dd416fc Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Sat, 20 Oct 2018 18:04:43 -0600
Subject: [PATCH 4/9] msm8974-common: sepolicy: Resolve system_server denials

* avc: denied { call } for scontext=u:r:system_server:s0
  tcontext=u:r:init:s0 tclass=binder permissive=1

Change-Id: Id2e10c8422df81453a666d23253a1ea6bb614c34
---
 sepolicy/common/system_server.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sepolicy/common/system_server.te b/sepolicy/common/system_server.te
index 3d0c927..9dd7d37 100644
--- a/sepolicy/common/system_server.te
+++ b/sepolicy/common/system_server.te
@@ -1,3 +1,4 @@
+binder_call(system_server, init)
 get_prop(system_server, alarm_boot_prop)
 
 allow system_server efs_file:dir search;
-- 
2.17.1

